kolibri_explore_plugin/assets/src/views/ChannelsPage.vue

```diff
@@ -166,12 +166,14 @@
     background-color: white;
   }
 
-  .header {
-    background: white;
+  .discovery-header {
+    height: $discovery-navbar-height;
```

Comment:
This makes the header size inconsistent with the template. Let's ask Joana to confirm this. At this point we need to start closing things.